content/api/cypress-api/custom-commands.md

```diff
@@ -738,10 +738,12 @@ const search = (term, options = {}) => {
   // chain off this function below
   return cy
     .log(`Searching for: ${term} `)
-    .route({
-      url: '/search/**',
-      response: `fixture:${fixture}`,
-      headers: headers,
+    .intercept('GET', '/search/**', (req) => {
+      req.reply({
+        statusCode: 200,
+        body: `fixture:${fixture}`,
+        headers: headers,
+      })
```

Comment:
Alternatively, we could've used [the shorthand syntax](https://docs.cypress.io/api/commands/intercept#req-reply-shorthand).

```suggestion
      req.reply(200, fixture, headers)
```